pr_agent/algo/utils.py

```diff
@@ -271,6 +271,13 @@ def load_yaml(review_text: str) -> dict:
 
 def try_fix_yaml(review_text: str) -> dict:
     review_text_lines = review_text.split('\n')
+
```

Comment:
If the malformed YAML read something like:
title: My:title

A better approach could be to quote the value? Those colons may be significant..